.github/workflows/rust.yml

```diff
@@ -44,16 +44,16 @@ jobs:
       matrix:
         rust: [msrv, latest]
         sys:
-          - os: ubuntu-latest-16-cores
-            target: x86_64-unknown-linux-gnu
-          - os: ubuntu-latest-16-cores
-            target: aarch64-unknown-linux-gnu
-          - os: macos-latest
-            target: x86_64-apple-darwin
-          - os: macos-latest
-            target: aarch64-apple-darwin
-          - os: windows-latest-8-cores
-            target: x86_64-pc-windows-msvc
+        - os: ubuntu-latest-16-cores
```

Comment:
@leighmcculloch I know we don't really follow this style, but isn't proper yaml style to add indent on new object start?
So it should be
```
sys:
__-os: ...
```
not 
```
sys:
- os: <...>
```
Asking because I personally find it more readable + my IDE auto-formats it like that 😆